### ISSUE TITLE
help

### ISSUE THREAD

**Issue by chanchallanghni** [chanchallanghni/chanchal#1]
help 

Want to send sensor data from your Arduino uno R3  to your Android application using bluetooth hc05

The project VITALITY **Connet.Collaborate.Care**has used the WBAN technology to create a portable device (made up of sensors)that can monitor patient’s health and send the patient data wirelessly to the base station and provide the access of data through an android application. 
We have three modules:
1)      Configuration and interfacing of sensors (temperature sensor (LM35) and pulse rate sensor) with Arduino UNO using its Arduino Software (IDE).

2)      Designing and development of Android Application using Android Studio and Database using Firebase for real time transferring of data.

3)       Interconnection between the sensor and application by using Bluetooth module (HC-05).

We are done with the first two modules and having problem in the implementation of the third module may be the problem is with the application or some issues with hardware we are confused and facing problem in detecting where the link drops.

Basically issue is with the transferring of data from sensors to application using Bluetooth .
